ynr/apps/uk_results/management/commands/import_2010_parl_results.py

```diff
@@ -87,16 +108,18 @@ def create_results(self, ballot, candidates):
         if changed:
             self.stdout.write("Recorded new results!")
             self.results_added += 1
-            resultsbot = ResultsBot()
-            resultsbot._mark_candidates_as_winner(resultset)
-
-            LoggedAction.objects.create(
-                user=resultset.user,
-                action_type=ActionType.ENTERED_RESULTS_DATA,
-                source=resultset.source,
-                ballot=resultset.ballot,
-                edit_type=EditType.BOT.name,
+            winner = (
+                resultset.candidate_results.order_by("-num_ballots")
+                .first()
+                .membership.person
+            )
+            resultsrecorder = RecordBallotResultsHelper(
+                ballot=ballot, user=ResultsBot().user
             )
+            resultsrecorder.mark_person_as_elected(
```

Comment:
@symroe think the usage of this would be one of the main things to check. I dont want to end up creating a load of duplicate events/logged actions. Although with changes made in the PR I dont think it would do